docs/components/copy.md

```diff
@@ -151,22 +151,61 @@ Number of characters to truncate at. Defaults to `8`.
 />
 ```
 
+If you specify `'auto'` for the `truncationLimit` the text will only truncate if there is not enough space for it to be fully displayed.
+
+<KCard>
+  <div class="copy-flexed">
```

Comment:
suggestion: it isn't evident that the KCopy adjusts to the parent size without resizing the window. Could you replicate the behaviour we have in [KTruncate component default slot docs](https://alpha--kongponents.netlify.app/components/truncate.html#default) with resizable container?